/**:
  ros__parameters:
    model_name: gcn
    input_dim: 2
    hidden_dim: 128
    output_dim: 2 
    num_layers: 4
    pool_method: mean ## mean or max
    debug_mode: false
    